### PR TITLE
Slashed tokens

### PR DESCRIPTION
**Pull request by r-czajkowski**
Depends on: #1321 
Closes: #1378 

This PR adds integration with smart contracts to fetching slashed tokens.

obraz

### CODE REVIEW

File: solidity/dashboard/src/components/SlashedTokensList.jsx [hunk 1]
```diff
@@ -1,47 +1,43 @@
...
-        </li>
+        {slashedTokens.map(renderSlashedTokenItem)}
       </ul>
     </div>
   )
 }
 
+const renderSlashedTokenItem = (item) => <SlashedTokeItem key={item.id} {...item} />
+
+const SlashedTokeItem = React.memo(({ amount, date, event, groupPublicKey }) => (
```
Comment (lukasz-zimnoch) [inline solidity/dashboard/src/components/SlashedTokensList.jsx]: Typo `SlashedTokeItem`

File: solidity/scripts/misbehaved-groups.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+async function registerNewGroup (keepRandomBeaconOerator, accounts) {
+  const groupPubKey = crypto.randomBytes(32);
+  await keepRandomBeaconOerator.registerNewGroup(groupPubKey)
+  await keepRandomBeaconOerator.setGroupMembers(groupPubKey, [accounts[1], accounts[2], accounts[1]]);
+}
+  
+module.exports = async function() {
+    try {
+        const accounts = await getAccounts();
+        const keepRandomBeaconOerator = await Operator.deployed();
```
Comment (lukasz-zimnoch) [inline solidity/scripts/misbehaved-groups.js]: Typo `keepRandomBeaconOerator`

File: solidity/scripts/misbehaved-groups.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+function getAccounts() {
+  return new Promise((resolve, reject) => {
+      web3.eth.getAccounts((error, accounts) => {
+        resolve(accounts);
+      });
+  });
+}
+
+async function registerNewGroup (keepRandomBeaconOerator, accounts) {
+  const groupPubKey = crypto.randomBytes(32);
```
Comment (lukasz-zimnoch) [inline solidity/scripts/misbehaved-groups.js]: Please avoid abbreviations: `groupPubKey` -> `groupPublicKey`

File: solidity/scripts/misbehaved-groups.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  await keepRandomBeaconOerator.registerNewGroup(groupPubKey)
+  await keepRandomBeaconOerator.setGroupMembers(groupPubKey, [accounts[1], accounts[2], accounts[1]]);
```
Comment (lukasz-zimnoch) [inline solidity/scripts/misbehaved-groups.js]: Is it intentional to put account 1 twice?

File: solidity/scripts/misbehaved-groups.js [hunk 5]
```diff
@@ -0,0 +1,41 @@
+/**
+ * Important: the KeepRandomBeaconOperatorRewardsStub
+ * and the TokenStakingSlashingStubs contract should be deployed to the network!
+ * 
+ * This script:
+ *  - emits fake events (UnauthorizedSigningReported, RelayEntryTimeoutReported)
+ *  */ 
+
+// The KeepRandomBeaconOperatorRewardsStub contract should be deployed to the network
+const Operator = artifacts.require("../stubs/KeepRandomBeaconOperatorRewardsStub.sol");
```
Comment (lukasz-zimnoch) [inline solidity/scripts/misbehaved-groups.js]: Please use the full name: `KeepRandomBeaconOperator`. `Operator` is a bit unclear.

File: solidity/scripts/misbehaved-groups.js (hunk 4 above)
Comment (r-czajkowski) [reply solidity/scripts/misbehaved-groups.js]: Yep. For testing when a given operator is in a given group many times.

**Pull request merged**